pages/Messages/Identifiers.md

```diff
@@ -0,0 +1,91 @@
+---
+name: Identifiers
+route: /Messages/Identifiers
+menu: Messages
+---
+
+# Messages Identifiers
+
+## Specification Status
+
+| Version | Status |
+---------- | ---------
+| 0.1     | Tentative |
+
+## Purpose
+
+1. Describe a means of uniquely identifying DSNP content messages
+1. Describe potential extensions of identifiers for the future
+
+## Assumptions
+
+* All assumptions from [DSNP Messages](/Messages/Overview)
+
+## Identifiers
+
+DSNP message identifiers at minimum consist of a string with three parts, a protocol, a user identifier and a content hash, like so:
+
+```
+dsnp://0123456789abcdef/0123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef
+```
+
+In this example, `dsnp://` is the protocol indicating that the latest version of this specification should be used.
+`0123456789abcdef` is the identifier representing a user as described in the [Identity Registry](/Identity/Registry) specification.
+`0123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef` is the hash of the content as described in the [Messages Serialization](/Messages/Serialization) specification.
+
+### Protocol
+
+The protocol section of an identifier can be used to indicate which version of the DSNP specification should be used to identify content.
+If present, any number after the string `dsnp` should indicate the specific version.
+For example, `dsnp1.2.1://` would indicate that version 1, subversion 2, patch release 1 of the specification should be used.
```

Comment:
Hmm... I wonder if we want to allow this level of detail. Especially as this identifier is going to be stored in the announcement (inReplyTo), it means that we cannot go back and change it.

I think we should just lock the version and only bump for breaking changes aka: `dsnp://` -> `dsnp2://` and require it be that way on all v2.

A non-breaking change would be like supporting `@username` instead of the DSNP Id. (a breaking change would be requiring 0x prefixed hex values).